cmd/rosetta/main.go

```diff
@@ -15,6 +15,14 @@ import (
 )
 
 func main() {
+	// This is a workaround, so that TCP sockets opened by rosetta (as a TCP client) to read data from the observer can be reused more easily,
```

Comment:
Perhaps we should also investigate the closing of HTTP requests on Observer's side, just to ensure it's all right.

Reply:
Please review a bit the explanation - is it correct?

Also see:
 - https://github.com/golang/go/issues/16012
 - https://vincent.bernat.ch/en/blog/2014-tcp-time-wait-state-linux